flutter/ios/ci_scripts/ci_post_clone.sh

```diff
@@ -62,18 +103,27 @@ cd "$MC_REPO_HOME"/flutter && flutter precache --ios
 
 echo "$MC_LOG_PREFIX ========== Build app =========="
 export BAZEL_OUTPUT_ROOT_ARG=--output_user_root=$MC_BUILD_HOME/bazel
+export WITH_TFLITE=0
+export WITH_APPLE=1
 
 echo "$MC_LOG_PREFIX Build backend and Flutter packages"
 # Remember to update the next line if make commands are changed.
 cd "$MC_REPO_HOME" && time make flutter/prepare && time make flutter/ios
 
-if [ "$CI_XCODEBUILD_ACTION" = "build-for-testing" ]; then
-  cd "$MC_REPO_HOME"/flutter && flutter build ios --config-only integration_test/first_test.dart
-else
-  cd "$MC_REPO_HOME"/flutter && flutter build ios --config-only --dart-define=official-build="$OFFICIAL_BUILD"
+if [ $runner = $XCODE_CLOUD ]; then
+  if [ "$CI_XCODEBUILD_ACTION" = "build-for-testing" ]; then
+    cd "$MC_REPO_HOME"/flutter && flutter build ios --config-only integration_test/first_test.dart
+  else
+    cd "$MC_REPO_HOME"/flutter && flutter build ios --config-only --dart-define=official-build="$OFFICIAL_BUILD"
+  fi
 fi
 
 cd "$MC_REPO_HOME"/flutter/ios && pod install
 
+if [ $runner = $GITHUB_ACTIONS ]; then
+  open -a Simulator
+  cd "$MC_REPO_HOME" && make flutter/test
```

Comment:
By the way, `make flutter/test` also runs unit tests.
I guess there's no real harm in running them, they are quite fast, but unit test are platform-independent, so there is no difference between running them here and running them in the `Unit tests / flutter` github workflow.

Reply:
Ideal case there should be no difference, but they are run on 2 different OS (Mac and Linux) so let it run on both would be better.